src/index.tsx

```diff
@@ -1018,10 +1019,13 @@ export class App extends React.Component<{}, AppState> {
                 draggingElement.isSelected = true;
               }
 
-              this.setState({
-                draggingElement: null,
-                elementType: "selection"
-              });
+              if (!e[META_KEY]) {
+                // Doesn't reset selection when the user holds cmd or ctrl
```

Comment:
This comment doesn't make sense. It suggests the exact opposite of what the next statement is doing. You should move the comment above the `if` statement  --- also, would change the comment to `don't reset shape tool if user holds META_KEY`